### PR TITLE
Add certificate issuer and kid to certlogic payload

### PR DESCRIPTION
**Pull request by M-Wong**

### CODE REVIEW

File: src/main/java/ch/admin/bag/covidcertificate/sdk/core/models/certlogic/CertLogicData.kt [hunk 1]
```diff
@@ -30,5 +30,7 @@ internal data class CertLogicHeaders(
 	val iat: String?,
 	val exp: String?,
 	val isIsLight: Boolean?, //Must be isIsLight because the Json Serializer strips one "is"
-	val mode: String?
+	val mode: String?,
+	val iss: String? = null,
+	val kid: String? = null,
```
Comment (ubamrein) [inline src/main/java/ch/admin/bag/covidcertificate/sdk/core/models/certlogic/CertLogicData.kt:35]: Should it be kidBase64?

File: src/main/java/ch/admin/bag/covidcertificate/sdk/core/verifier/CertificateVerifier.kt [hunk 2]
```diff
@@ -228,11 +228,13 @@ class CertificateVerifier {
 					val nationalRulesVerifier = NationalRulesVerifier()
 					val issuedAt = certificateHolder.issuedAt?.prettyPrint(DEFAULT_DISPLAY_RULES_TIME_FORMATTER)
 					val expiredAt = certificateHolder.expirationTime?.prettyPrint(DEFAULT_DISPLAY_RULES_TIME_FORMATTER)
+					val issuer = certificateHolder.issuer
+					val kid = certificateHolder.kidBase64
```
Comment (ubamrein) [inline src/main/java/ch/admin/bag/covidcertificate/sdk/core/verifier/CertificateVerifier.kt:232]: Same here?

File: src/main/java/ch/admin/bag/covidcertificate/sdk/core/models/certlogic/CertLogicData.kt (hunk 1 above)
Comment (M-Wong) [reply src/main/java/ch/admin/bag/covidcertificate/sdk/core/models/certlogic/CertLogicData.kt:35]: Left as `kid` as iOS does the same: https://github.com/admin-ch/CovidCertificate-SDK-iOS/pull/141/files

File: src/main/java/ch/admin/bag/covidcertificate/sdk/core/verifier/CertificateVerifier.kt (hunk 2 above)
Comment (M-Wong) [reply src/main/java/ch/admin/bag/covidcertificate/sdk/core/verifier/CertificateVerifier.kt:232]: Left as `kid` as iOS does the same: https://github.com/admin-ch/CovidCertificate-SDK-iOS/pull/141/files

**Review by benz-ubique — APPROVED** [review]

**Pull request merged**